src/Illuminate/Auth/SessionGuard.php

```diff
@@ -904,6 +919,16 @@ public function getSession()
         return $this->session;
     }
 
+    /**
+     * Get the timebox instance used by the guard.
+     *
+     * @return \Illuminate\Support\Timebox
+     */
+    public function getTimebox()
```

Comment:
It looks like this getter was added in the guard just because of tests. IMO this should be removed from this class and the test should be refactored so that it does not need this getter (changing "production" code just because of tests does not seem like a good idea).

Reply:
Thanks for the comment!

I was looking at the other tests and how that was done. The getSession() method which is also a getter on the SessionGuard class is also only used in the tests.

I somehow agree with you about creating code to only satisfy the tests. If there were checks in the production code that checked if you were running tests and then behaved differently I would also make an objection. But changing the production code to be more testable is for me one of the primary reasons for writing tests, so dont think that is a bad thing.

I will resolve the conversion now, but let me know if you disagree.

Reply:
You already know the timebox instance in the test as you are passing it through the constructor to the guard so I'm not sure how adding a getter makes this "more testable". I don't have a strong preference either way, but that's just my 2 cents.